test/command/suite/select/function/snippet/invalid/too_long_key.actual

```diff
@@ -0,0 +1,37 @@
+table_create Entries TABLE_NO_KEY
```

Comment:
`.actual` -> `.expected`.

Reply:
I have renamed and modified the test contents.